docs/source/workflows/mcp/mcp-client.md

```
@@ -187,34 +187,31 @@ Once configured, a Pydantic input schema will be generated based on the input sc
+    mcp_tool_name: calculator_multiply
+    description: "Returns the product of two numbers"
 ```
 
 To run the simple calculator workflow using remote MCP tools, follow these steps:
-1. Start the remote MCP server, `mcp-server-time`, by following the instructions in the `examples/MCP/simple_calculator_mcp/deploy_external_mcp/README.md` file. Check that the server is running by running the following command:
-```bash
-docker ps --filter "name=mcp-proxy-nat-time"
-```
-Sample output:
+1. Start the example remote MCP server.
 ```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)